full-moon/src/tokenizer.rs

```diff
@@ -95,6 +85,8 @@ pub enum TokenizerErrorType {
     UnclosedComment,
     /// An unclosed string was found
     UnclosedString,
+    /// An unexpected #! was found
+    UnexpectedShebang,
```

Comment:
Put this above UnclosedString, ABC order.

Reply:
`Unexpected..` come after `Unclosed...`, though I could sort on the second word.

Reply:
I just misread it, oops.